infoblox/resourceIPAllocation.go

```diff
@@ -112,16 +113,13 @@ func resourceIPAllocationGet(d *schema.ResourceData, m interface{}) error {
 
 	log.Printf("[DEBUG] %s:Reading the required IP from  network block", resourceIPAllocationIDString(d))
 
-	networkViewName := d.Get("network_view_name").(string)
 	tenantID := d.Get("tenant_id").(string)
-	ipAddr := d.Get("ip_addr").(string)
 	cidr := d.Get("cidr").(string)
-	macAddr := d.Get("mac_addr").(string)
 	connector := m.(*ibclient.Connector)
 
 	objMgr := ibclient.NewObjectManager(connector, "terraform", tenantID)
 
-	_, err := objMgr.GetFixedAddress(networkViewName, cidr, ipAddr, macAddr)
+	_, err := objMgr.GetHostRecord(d.Id())
```

Comment:
Need a short description why we are using host record rather than fixed address.